joomla/joomla-cms
# Fix for #7388 .. Fix some more undefined variables in core.js, for "strict mode"

Here is fix for #7388
For test please follow instruction from that issue :wink: 

---
# Testing instructions

Download joomla from staging: https://github.com/joomla/joomla-cms/archive/staging.zip

Try to install:
- fill the details of the first screen and click Next
- installation will halt.
- check your browser console you will see the error
